src/ProductConstructionService/ProductConstructionService.Api/Dockerfile

```diff
@@ -43,13 +49,26 @@ COPY ["src/ProductConstructionService", "./ProductConstructionService"]
 
 RUN dotnet build "./ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj" -c $BUILD_CONFIGURATION -o /app/build --no-restore
 
+# Build Angular app
+FROM mcr.microsoft.com/devcontainers/typescript-node:dev-20 AS angular
+WORKDIR /maestro-angular
+COPY ["src/Maestro/maestro-angular", "."]
+ENV NODE_OPTIONS=--openssl-legacy-provider
+RUN npm install
+RUN npm run build -- --prod
```

Comment:
just checking, this should always be prod, even when running in staging?

Reply:
Yes, it matches what `maestro-angular.proj` does